zilencer/models.py

```diff
@@ -587,3 +587,26 @@ def has_stale_audit_log(server: RemoteZulipServer) -> bool:
         return True
 
     return False
+
+
+class RemotePushDevice(models.Model):
```

Comment:
This should get comments for the fields, like one before realm/remote_realm explaining why we have both columns.

Reply:
Comments added.